ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -58,10 +60,34 @@ import { ReactComponent as wearable } from 'assets/icons/DAR/wearable.svg';
 import { TwoFactorAuthModal } from './two-factor-auth-modal';
 
 const styles = reactStyles({
-  headerFlexColumn: {
+  initialRegistrationOuterHeader: {
     marginLeft: '3%',
     width: '50%',
   },
+  initialRegistrationHeaderRW: {
+    textTransform: 'uppercase',
+    margin: '1em 0 0 0',
+  },
+  initialRegistrationHeaderDAR: {
+    height: '30px',
+    width: '302px',
+    fontFamily: 'Montserrat',
+    fontSize: '22px',
```

Comment:
should fontSize be a number rather than px?

Reply:
done